addons/website_forum/views/forum.xml

```diff
@@ -3,7 +3,7 @@
     <data>
         <!-- MAIN FORUM MENU -->
         <menuitem name="Forum" id="menu_website_forum"
-            parent="mail.mail_feeds_main" sequence="50"/>
+            parent="mail.mail_feeds_main" sequence="50" groups="base.group_no_one"/>
```

Comment:
Note that you have to access the back-end of forum to tweak the karma stuff. Marketing something maybe ?